openpecha/formatters/ocr/ocr.py

```diff
@@ -645,7 +646,7 @@ def set_base_meta(self, image_group_id, base_file_name, word_confidence_list):
               "ocr_word_mean_confidence_index": statistics.mean(word_confidence_list)
             }
     
-    def create_opf(self, data_provider, pecha_id=None, opf_options = {}, ocr_import_info = {}):
```

Comment:
`pecha_id` should stay as an optional argument, the new value should only be computed if the `pecha_id` argument is `None`